Ghost-Hunter/Assets/Scripts/PlayerMovement.cs

```diff
@@ -33,6 +36,10 @@ void Start()
     // Update is called once per frame
     void Update()
     {
+        bool horizontalButtonPressed = Input.GetButton("Horizontal");
+        bool verticalButtonPressed = Input.GetButton("Vertical");
+        
+        main.transform.position = new Vector3(myTransform.position.x, myTransform.position.y, -10);
```

Comment:
```suggestion
        main.transform.position = new Vector3(myRigidbody.position.x, myRigidbody.position.y, -10);
```
MyTransform doesn't exist, but if you change it to myRigidbody it should work fine